apps/dashboard/src/@/components/blocks/wallet-address.tsx

```diff
@@ -62,10 +65,9 @@ export function WalletAddress(props: {
             props.className,
           )}
         >
-          <WalletAvatar
-            address={props.address}
-            profiles={profiles.data || []}
-          />
+          {address && (
```

Comment:
doesn't this assume address will be defined so adding a `string | undefined` is redundant above?